pkg/operator/credentialsrequest/credentialsrequest_controller.go

```diff
@@ -239,9 +239,75 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 		return err
 	}
 
+	// Monitor the cloud-credential-operator-config configmap to check if operator is re-enabled.
+	// Check if attempts were made to delete the credentials requests when the operator was disabled.
+	// We have to have to reconcile all those credential requests.
+	configmapMapFn := handler.ToRequestsFunc(
+		func(a handler.MapObject) []reconcile.Request {
+			// Iterate all CredentialsRequests to determine if any of them have DeletionTimestamp set
+			log.Debug("checking for credentials requests marked for cleanup when operator was disabled")
+			crs := &minterv1.CredentialsRequestList{}
+			mgr.GetClient().List(context.TODO(), crs)
+			requests := []reconcile.Request{}
```

Comment:
This should also be "var requests []reconcile.Request", according to goland.

Reply:
I ended up with 

```go
	// allCredRequestsMapFn simply looks up all CredentialsRequests and requests they be reconciled.
	allCredRequestsMapFn := handler.ToRequestsFunc(
		func(a handler.MapObject) []reconcile.Request {
			log.Info("requeueing all CredentialsRequests")
			crs := &minterv1.CredentialsRequestList{}
			err := mgr.GetClient().List(context.TODO(), crs)
			var requests []reconcile.Request
			if err != nil {
				log.WithError(err).Error("error listing all cred requests for requeue")
				return requests
			}
			for _, cr := range crs.Items {
					requests = append(requests, reconcile.Request{
						NamespacedName: types.NamespacedName{
							Name:      cr.Name,
							Namespace: cr.Namespace,
						},
					})
			}
			return requests
		})
```

And I think we can just re-use the same map function, we both just want to requeue everything.